package/cfg/config.yaml

```diff
@@ -243,6 +255,12 @@ target_mapping:
     - "controlplane"
     - "etcd"
     - "policies"
+  "rke-cis-1.6-permissive":
+    - "master"
+    - "node"
+    - "controlplane"
+    - "etcd"
+    - "policies"
```

Comment:
We should also add the profiles to the "version_mapping" map in this file: https://github.com/rancher/security-scan/blob/master/package/cfg/config.yaml#L191

Reply:
added it in, changed default profile to CIS 1.6 for k8s 1.16 onwards as per discussion offline with you and @cloudnautique

Reply:
Just putting a note here: we will have to add new scanprofile and benchmark CRs in the chart for cis-1.6 profiles.